forms-flow-idm/migration/migrate-7.2.0.py

```diff
@@ -0,0 +1,33 @@
+
+import argparse
+from utils import get_access_token, get_client_id, add_client_roles, update_client_role, role_changes_710
+from typing import List
+
+# Roles to be added, removed, or updated for the forms-flow-web client
+roles_to_add, roles_to_remove, roles_to_update = role_changes_710()
```

Comment:
Is the role changes for `7.1.0` and `7.2.0` same? If yes, do we need this migrate script?